### DISCUSSION TITLE
Bug: Icons not showing for Qt

### DISCUSSION THREAD

**Discussion by gruenich** [audacious-media-player/audacious#70] [General]
For a couple of week my audacious no longer shows icons (only replacement text in my local language German):

![image](https://github.com/audacious-media-player/audacious/assets/4538230/dcafac42-16d1-4b44-bf16-70a17a369a04)

This happens with the package from openSuse and occurred the first time a couple of weeks, maybe months ago. First I thought it would be a packaging bug, but I downloaded audacious source myself and built it (`./configure --disable-gtk --disable-gtk3 --enable-qt && make`) and the problem remained. Further, I checked out audaciuous 4.0 and had the same issue.

I started audacious from my terminal. I did not get any warning that files are not found or similar. Is there a way to enable logging / get some warnings?

What should I do? Can I provide further information?

**Comment by radioactiveman** [comment]
Is the Qt SVG package installed?

- https://wiki.archlinux.org/title/Qt#Icon_theme_is_not_applied
- https://software.opensuse.org/package/libqt5-qtsvg

**Reply by gruenich** [reply]
I have the Qt 5 SVG package installed. Out of curiosity, I installed also Qt 6 SVG (openSuse calls it `libQt6Svg6`) and now I see Audocious with icons for the first time since weeks. This fixed both the version from openSuse's Audacious RPM and my self-compiled binary using Autotools.

**Comment by gruenich** [comment]
Should Audacious print out a warning (or write to the journal) that it is missing the Qt SVG library?

**Comment by radioactiveman** [comment]
Audacious does not specify explicitly to use SVG icons for the toolbar. We only specify the icon name.
Such warning is therefore not possible on application side.

In my opinion Qt itself should take care of it. You may report this issue on their bug tracker.

And OpenSuse should add the SVG package as dependency to Audacious, similar to [Arch Linux](https://gitlab.archlinux.org/archlinux/packaging/packages/audacious/-/commit/18a36707f2290d43da1f7b3fea1557328be96e0c)  and [Fedora](https://src.fedoraproject.org/rpms/audacious/c/3a03874cee3f4ecca80085c028133fbc9e152cb3).

**Reply by gruenich** [reply]
> And OpenSuse should add the SVG package as dependency to Audacious, similar to [Arch Linux](https://gitlab.archlinux.org/archlinux/packaging/packages/audacious/-/commit/18a36707f2290d43da1f7b3fea1557328be96e0c) and [Fedora](https://src.fedoraproject.org/rpms/audacious/c/3a03874cee3f4ecca80085c028133fbc9e152cb3).

I did so and my adjustment to the package got accepted. Thank you for your help!
